src/components/AmountFilter.jsx

```diff
@@ -1,57 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { InputLabel, Grid, Slider } from '@mui/material';
-import { Remove as RemoveIcon } from '@mui/icons-material';
+import { InputLabel, Stack, Slider } from '@mui/material';
 import AmountTextField from './AmountTextField';
 
 export default function AmountFilter(props) {
-  const { min, max, onMinChange, onMaxChange } = props;
-  const error = max > 0 && max < min;
+  const { min, onMinChange } = props;
+  const error = min < 0;
 
   return (
-    <Grid
-      container
-      justifyContent="space-between"
-      sx={{ maxWidth: 300, m: 'auto' }}>
-      <Grid item>
+    <Stack alignItems="center" sx={{ maxWidth: 300, m: 'auto' }}>
+      <>
         <InputLabel>Min</InputLabel>
         <AmountTextField
           error={error}
           value={min || ''}
           onChange={(e) => onMinChange(e.target.value)}
           placeholder="No min"
         />
```

Comment:
I wonder if we should get rid of this too 🤔

Reply:
and just leave the slider?

Reply:
Hmm, my concern is around the space it takes :P.

What if we change the Stack direction, put the text field to the right, and use the `variant="standard"` instead?